content/blog/2016/2016-05-04-the-state-of-JAM

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "The State of Jenkins Area Meetup (JAM)"
+tags:
+- meetup
+- JAM
+- event
+author: alyssat
+---
+
+NOTE: _This is a guest post by link:https://github.com/alyssat[Alyssa Tong].
```

Comment:
yes, the content is mine..you can tell by the sucky writing. Hannah just
posted it for me. This is how we've done it in the past and it has worked
out well for us. But if this is an issue we'll fix it.

On Thu, May 5, 2016 at 10:50 AM, hinman55 <notifications@github.com> wrote:

> In content/blog/2016/2016-05-04-the-state-of-JAM
> <https://github.com/jenkins-infra/jenkins.io/pull/247#discussion_r62225070>
> :
>
> > @@ -0,0 +1,44 @@
> > +---
> > +layout: post
> > +title: "The State of Jenkins Area Meetup (JAM)"
> > +tags:
> > +- meetup
> > +- JAM
> > +- event
> > +author: alyssat
> > +---
> > +
> > +NOTE: _This is a guest post by link:https://github.com/alyssat[Alyssa Tong].
>
> yes that is the case! i'll delete the 'guest post' blurb
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/247/files/83fe922de648f2d4fced8e81a9e6faf63a500ad3#r62225070>
>